### DISCUSSION TITLE
Increase HttpClient timeout

### DISCUSSION THREAD

**Discussion by mrob1600** [dotnet/yarp#502] [Q&A] [answered]
What approach can be used to override the default HttpClient timeout?

**Comment by Tratcher** [comment, marked as answer]
That's not exposed yet, see https://github.com/microsoft/reverse-proxy/issues/319

Edit, unless you're only using the IHttpProxy API?
https://github.com/microsoft/reverse-proxy/blob/f853abc0e0d64f4c31ee84870dfcbefee5bb5689/src/ReverseProxy/Service/Proxy/RequestProxyOptions.cs#L25

**Reply by mrob1600** [reply]
Okay thanks. No not using IHttpProxy API.
